.github/workflows/ci.yaml

```diff
@@ -0,0 +1,514 @@
+          source ./scripts/ci/lib.sh
+          get_genesis_dump
+
+      - name: Make bundle
+        run: image/scanner/rhel/create-bundle.sh image/scanner image/scanner/rhel
+
+      - name: Archive the bundle to preserve permissions
+        run: tar -cvzf bundle.tgz image/scanner/rhel
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: scanner-bundle
+          path: bundle.tgz
+
+  generate-scanner-db-bundle:
+    runs-on: ubuntu-latest
+    needs:
+      - generate-db-dump
+    if: |
+      always() &&
+      (needs.generate-db-dump.result == 'success' || needs.generate-db-dump.result == 'skipped')
+    container:
+      image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.61
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: ./.github/actions/job-preamble
+
+      - uses: actions/download-artifact@v3
+        if: |
+          github.event_name != 'pull_request' ||
+          contains(github.event.pull_request.labels.*.name, 'generate-dumps-on-pr')
+        with:
+          name: db-dump
+          path: /tmp/postgres
+
+      - name: Get db dump
+        run: |
+          source ./scripts/ci/lib.sh
+          get_db_dump
+
+      - name: Make db bundle
+        run: image/db/rhel/create-bundle.sh image/db image/db/rhel
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: scanner-db-bundle
+          path: image/db/rhel
+
+  build-images:
+    env:
+      QUAY_RHACS_ENG_RO_USERNAME: ${{ secrets.QUAY_RHACS_ENG_RO_USERNAME }}
+      QUAY_RHACS_ENG_RO_PASSWORD: ${{ secrets.QUAY_RHACS_ENG_RO_PASSWORD }}
+      QUAY_RHACS_ENG_RW_USERNAME: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
+      QUAY_RHACS_ENG_RW_PASSWORD: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
+      QUAY_STACKROX_IO_RW_USERNAME: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
+      QUAY_STACKROX_IO_RW_PASSWORD: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
+    runs-on: ubuntu-latest
+    needs:
+      - generate-scanner-bundle
+      - generate-scanner-db-bundle
+    if: always() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled')
+    container:
+      image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.61
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: ./.github/actions/job-preamble
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: scanner-bundle
+
+      - name: Unpack bundle
+        run: |
+          tar xvzf bundle.tgz
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: scanner-db-bundle
+          path: image/db/rhel
+
+      - name: Build scanner image
+        run: |
+          docker build -t scanner:"$(make --quiet tag)" -f image/scanner/rhel/Dockerfile image/scanner/rhel
```

Comment:
want to throw a `--no-print-directory` in all these calls to `make tag`?

Reply:
I'm down, though I'm not entirely sure if adding that flag will do anything since we aren't switching directories